content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx

```diff
@@ -8,23 +8,21 @@ preview: true
 
 <TopBlock>
 
-Prisma allows configuration of database indexes, unique constraints and primary key constraints through the `extendedIndexes` preview feature in version `3.5.0` and later. This provides extended syntax options in the Prisma schema language for configuring the length of indexes (MySQL only), the sort order (all databases), the index type (PostgreSQL only) and to configure indexes as clustered or non-clustered (SQL Server only).
+Prisma allows configuration of database indexes, unique constraints and primary key constraints. This can be enabled through the `extendedIndexes` Preview feature in versions `3.5.0` and later, and is in General Availability in versions `4.0.0` and later.
```

Comment:
A couple of suggestions for a rewrite to avoid passive voice, and to be more direct:
"You can enable this through the `extendedIndexes` Preview feature..."
"Enable this through the `extendedIndexes` Preview feature..."

Also I didn't know that we are capitalizing "Preview" but I do like it and it feels necessary, to make it more obvious. We should add to the Style Guide, along with a million other things.

Reply:
I suggest "with" instead of "through". I think it's clearer for second-language speakers.

Reply:
Do we need to capitalize preview? On a more general point, I've written stabilization notices differently (and yes, I had "preview" down as lower case), and I think moving forwards we should have a standard way of expressing it. Not necessarily my way! We write about this often enough that we could, as a group, come up with an ideal wording.

Reply:
We do capitalize 'Preview' everywhere - not sure whether I like it but that's the convention currently.

Thanks, I've rewritten to use active voice and "with" - still reads as clunky so feel free to have another crack at it/let me know what you've used elsewhere. Agree that we should come up with a wording.